foucaultview.cpp

```diff
@@ -74,10 +74,27 @@ void foucaultView::showContextMenu(QPoint pos)
     QMenu myMenu;
     myMenu.addAction("Save Ronchi image",  this, &foucaultView::saveRonchiImage);
     myMenu.addAction("Save Foucault Image", this,  &foucaultView::saveFoucaultImage);
+    QAction  *showAllRonchi = new QAction("Show all Selected Wave Fronts using Ronchi");
+    connect (showAllRonchi, &QAction::triggered,this, &foucaultView::showSelectedRonchiImages);
+    myMenu.addAction(showAllRonchi);
 
     // Show context menu at handling position
     myMenu.exec(globalPos);
 }
+
+void foucaultView::showSelectedRonchiImages(){
+
+    surfaceAnalysisTools *saTools = surfaceAnalysisTools::get_Instance();
+    QList<int> list = saTools->SelectedWaveFronts();
+
+    QList<wavefront*> wfs;
+
+    for (int i = 0; i < list.size(); ++i){
```

Comment:
I like to use size_t for arrays and lists as this is what size() returns. 
low importance
Shall compile same.
```suggestion
    for (size_t i = 0; i < list.size(); ++i){
```